compiler/optimizer/FieldPrivatizer.cpp

```diff
@@ -85,6 +85,15 @@ TR_FieldPrivatizer::TR_FieldPrivatizer(TR::OptimizationManager *manager)
 
 int32_t TR_FieldPrivatizer::perform()
    {
+
+   // TODO: need to confirm places calling opCodeFor* APIs are doing the right
```

Comment:
Can you create an issue for this and remove the `TODO`?

Reply:
removed the `TODO`. this is actually already an issue tracking this. I am putting the issue in the comment.